doc/script_commands.txt

```diff
@@ -2260,6 +2260,56 @@ array, shifting all the elements beyond this towards the beginning.
 
 ---------------------------------------
 
+*inarray <array name>,<value>;
+
+This command will return the index of the value inside the array.
+It will return -1 if the value not found.
+
+	setarray @array[0], 100, 200, 300, 400, 500, 600;
+	// So we have made @array[]
+	
+	inarray(@array, 200);
+	//return 1 because the number 200 is the index 1 for the array
+	//another way to say it that @array[1] == 200
+	
+	.@index = inarray(@array, 600);
+	//.@index is now 5 because @array[5] == 600
+	
+	inarray(@array, 100);
+	//return 0 because @array[0] == 100
+	
+	inarray(@array, 800);
+	//return -1 because 800 is not an element of the array @array
+
+For more details, see the sample in 'doc/sample/inarray.txt'.
+
+---------------------------------------
+
+*countinarray <array name>{[<start index>]},<value1>{,<value2>...<valueN>};
```

Comment:
What about to return the number of matches of 2 arrays for dynamical script instead of `<value1>{,<value2>...<valueN>}`?

Reply:
thanks , i will .